api/src/main/java/org/openmrs/PatientProgram.java

```diff
@@ -398,4 +410,86 @@ public void setLocation(Location location) {
 		Collections.sort(sortedStates);
 		return sortedStates;
 	}
+
+	@Override
```

Comment:
Am starting to think that may be you might want to override these method to add @since annotations but delegate to the super class methods, what do you think @dkayiwa ?

Reply:
any decision on this. @dkayiwa ?

Reply:
I agree with @wluyima